moped-editor/src/components/GridTable/Filters.js

```diff
@@ -372,22 +356,22 @@ const Filters = ({
         <Grid
           item
           xs={6}
-          className={classes.filtersContainer}
+          sx={{ filtersContainerStyle }}
```

Comment:
@tillyw I had to do a triple take on why these styles weren't applying, and then I saw an extra set of curly braces in here that is preventing sx from accessing the key/values. There are a couple of other ones in here too. could you update? 🙏